### PR TITLE
🌱 Promote sbueringer to Cluster API maintainer

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:
@sbueringer has been doing an incredible job in this project and would like to nominate for promotion as project maintainer.

/hold for consensus and community meeting

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by dims** [comment]
YES!!!

**Comment by enxebre** [comment]
+1000

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -22,6 +22,7 @@ aliases:
   - CecileRobertMichon
   - enxebre
   - fabriziopandini
+  - sbueringer
   - vincepri
 
   # folks who can review and LGTM any PRs in the repo
```
Comment (sbueringer) [inline OWNERS_ALIASES:28]: Nit: I think you want to drop l.31 too

**Comment by CecileRobertMichon** [comment]
huge +1!! 🙌

**Comment by stmcginnis** [comment]
/lgtm

Very well earned.

**Comment by fabriziopandini** [comment]
As per the 16th of February office hours discussion,
/hold cancel
yay!

**Comment by vincepri** [comment]
Congrats!!

**Pull request merged**